components/AddEntryForm/index.js

```diff
@@ -0,0 +1,89 @@
+import { styled } from "styled-components";
+import { StyledCard } from "../StyledComponents/StyledCard";
+import { StyledSection } from "../StyledComponents/StyledSection";
+import Heading from "../PageHeading";
+import * as wanakana from "wanakana";
+
+function AddEntryForm({ handleAddEntry, createNewEntryList }) {
+  function handleFormSubmit(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+    const newEntry = Object.fromEntries(formData);
+
+    handleAddEntry({ newEntry });
+
+    event.target.reset();
+    event.target.enInput.focus();
```

Comment:
```suggestion
    event.preventDefault();
    const form = event.target;

    const formData = new FormData(form);
    const newEntry = Object.fromEntries(formData);

    handleAddEntry({ newEntry });

    form.reset();
    form.enInput.focus();
```
 I feel like this might add just a smidge of readability.